ludwig/modules/attention_modules.py

```diff
@@ -181,6 +185,11 @@ def forward(self, inputs, mask=None):
             hidden = layer(hidden, mask=mask)
         return hidden
 
+    @property
+    def output_shape(self) -> torch.Size:
+        return torch.Size([self.sequence_size, self.input_size])
```

Comment:
should this be hidden size instead of input size?

Reply:
Good catch! Fixed.